roles/comanage-registry/tasks/main.yml

```diff
@@ -7,9 +7,55 @@
     - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
     - "{{ ansible_distribution }}.yml"
     - "{{ ansible_os_family }}.yml"
+  tags:
+  - always
 
-# Include OS-independent installation tasks
-- include: install.yml
+# Tasks deploying COmanage framework. There is none conditional task.
+- name: Include OS-independent installation tasks
+  include_tasks:
+    file: install.yml
+    apply:
+      tags:
+      - install
+  tags:
+  - install
 
-# Include OS-independent configuration tasks
-- include: configure.yml
+# Tasks required for basic Framework functionality
+- name: Include OS-independent setup tasks
+  include_tasks:
+    file: setup.yml
+    apply:
+      tags:
+      - setup
+  tags:
+  - setup
```

Comment:
I'd suggest to merge setup tasks into configure to be consistent with other roles, e.g. ssp